internal/osbuildexecutor/runner-impl-aws-ec2.go

```diff
@@ -53,52 +95,133 @@ func (ec2e *awsEC2Executor) RunOSBuild(manifest []byte, store, outputDirectory s
 		}
 	}()
 
-	logrus.Info("Spinning up jobsite manager")
-	args := []string{
-		"--builder-host",
-		*si.Instance.PrivateIpAddress,
-		"--store",
-		store,
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute*10)
+	defer cancel()
+	if !waitForSI(ctx, *si.Instance.PrivateIpAddress) {
+		return nil, err
 	}
+	controlFile, err := os.Open(filepath.Join("/var/tmp/osbuild-composer", "control.json"))
+	if err != nil {
+		return nil, err
+	}
+	defer os.Remove(controlFile.Name())
+	_, _ = controlFile.Write([]byte(fmt.Sprintf("{\"exports\": [\"%s\"]}", exports[0])))
```

Comment:
This is a super silly concern but if exports contain any special for json it would break the build or even allow injection of data. But I guess exports are not user controlled? Out of paranoia I would still include a validator here that errors if exports is not e.g. `[a-z0-9]+` or something similar (probably too strict but if we can start with an allow list that would be ideal).